tapeagents/core.py

```diff
@@ -61,15 +61,15 @@ class PartialStep(BaseModel):
 
 
 class Observation(Step):
-    pass
+    kind: Literal["observation"] = "observation"
```

Comment:
Let's add `kind: Literal["define_me"] = "define_me"` to `Step`. AFter that we don't need to add kinds to all marker models (Observation, Thought, ...), kinds will be added to the actual step classes with payloads.